webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -204,19 +222,36 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
             </div>
           </Tooltip>
         </summary>
-        {changeNbItemsPerRow && hasItems && maxNbPlotsPerRow > 1 && (
-          <div
-            className={styles.nbItemsPerRowSlider}
-            data-testid="nb-items-per-row-slider"
-          >
-            <MinMaxSlider
-              maximum={-1}
-              minimum={-maxNbPlotsPerRow}
-              label="Plot Width"
-              onChange={handleResize}
-              defaultValue={-nbItemsPerRow}
-            />
-          </div>
+        {changeSize && hasItems && maxNbPlotsPerRow > 1 && (
+          <>
+            <div
+              className={styles.sizeSliders}
+              data-testid="nb-items-per-row-slider"
```

Comment:
Should we rename this id since it now holds two different sliders? Or maybe give each slider their own id 🤔